GHzDACs/ghz_fpga_server.py

```diff
@@ -2133,17 +2152,20 @@ def adc_bringup(self, c):
     #   - ADC boards: either one waveform (averaged) for whole run
     #                 or one demodulation packet for each repetition
 
-# Runners contain information for running a sequence on a particular board
+# Runners contain information for running a sequence on a particular board.
 #
-# other info these should provide:
-# - what to upload
-# - whether upload is optional (based on setup state)
-# - how many packets to expect, based on repetitions
-# - whether discard is possible
+# Other info these should provide:
+# - what to upload.
+# - whether upload is optional (based on setup state).
+# - how many packets to expect, based on repetitions.
+# - whether discard is possible.
+
+# Some helper methods.
+def _AssertHasJumpTable(dev):
```

Comment:
Done. I avoided changing the the class method names since some of these are overriding base class names.